inference-engine/src/mkldnn_plugin/memory_descs/cpu_memory_desc.h

```diff
@@ -8,12 +8,16 @@
 #include <ie_precision.hpp>
 #include "cpu_shape.h"
 #include "utils/general_utils.h"
+#include "cpu_types.h"
 
 namespace MKLDNNPlugin {
```

Comment:
`class MemoryDesc;`

`using MemoryDescPtr = std::unique_ptr<MemoryDesc>;`
`using MemoryDescConstPtr = std::unique_ptr<const MemoryDesc>;`

Add this alias usage below.